File: test/fsharpTests/CharTest/fromCodeTest.fs
testCase "fromCode(97)" 
<| fun _ -> 
    let expected = Some('a')
    Expect.equal expected (Char.fromCode 97) "error"
testCase "fromCode(-1)" 
<| fun _ -> 
    let expected = None
    Expect.equal expected (Char.fromCode -1) "error"
testCase "fromCode(256)" 
<| fun _ -> 
    let expected = None
    Expect.equal expected (Char.fromCode 256) "error"

File: test/ocamlTests/CharTest/fromCodeTest.ml
test "fromCode(97)" (fun () -> expect (Char.fromCode 97) |> toEqual (let open Eq in option char) Some('a')) ; 
test "fromCode(-1)" (fun () -> expect (Char.fromCode -1) |> toEqual (let open Eq in option char) None) ; 
test "fromCode(256)" (fun () -> expect (Char.fromCode 256) |> toEqual (let open Eq in option char) None) ; 
